### ISSUE TITLE
Problem with placement top

### ISSUE THREAD

**Issue by nadnevi** [yuku-t/jquery-textcomplete#125]
I'm having a problem using the placement: top option.  When I use it on my site, the menu shows up far below the text I'm typing.

![image](https://cloud.githubusercontent.com/assets/5778152/4496357/5fc3206e-4a64-11e4-825d-5a33f938a060.png)

Without the placement option, the menu shows up correctly.

I'm using the latest version of the plugin.  My target browser is Chrome, but I have the same problem with Firefox.  Here is a Fiddle of the problem: http://jsfiddle.net/diven/fgpLpLow/

Any ideas?  Thanks!

**Comment by mioo** [comment]
I have same issue. When I use jquery-textcomplete in twitter bootstrap modal, the placement very random.

**Comment by mioo** [comment]
I tried another autocomplete plugin and solved this issue.
You can find it here https://github.com/ichord/At.js .

**Comment by yoshinari-nomura** [comment]
I encountered the same issue.
As for the bootstrap modal window, I found a quick hack.
Add these options to textcomplete func (CoffeeScript):
```coffee
    zIndex: 10000
    listPosition: (position) ->
      this.$el.css(this._applyPlacement(position))
      this.$el.css('position', 'absolute')
      return this
```
`listPosition` is undocumented feature, but was introduced at #88

IMHO, since Modal changes dynamically its css `position` property to `fixed`,
jQuery `$(textarea_in_modal).offset()` returns a value far off the screen,
if the modal was opened after the current page was scrolled.
textcomplete-popup window seems to obey the parent (modal) and changes its `position` to `fixed`.
Using this offset, the popup window is opened far off the screen because it has `fixed` property.
